### ISSUE TITLE
selenium is not in the requirements

### ISSUE THREAD

**Issue by pvanhemmen** [Torantulino/Auto-GPT#1398]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

selenium is imported but it's not in the requirements.txt

Try running `python -m autogpt`

### Current behavior 😯

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/paulvanhemmen/Python/Projects/autogpt/autogpt/__main__.py", line 12, in <module>
    from autogpt import commands as cmd
  File "/Users/paulvanhemmen/Python/Projects/autogpt/autogpt/commands.py", line 7, in <module>
    from autogpt.web import browse_website
  File "/Users/paulvanhemmen/Python/Projects/autogpt/autogpt/web.py", line 2, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
```

### Expected behavior 🤔

selenium should be in the requirements.txt

**Comment by darkcount2011** [comment]
Use the following:

 pip install selenium
pip install webdriver-manager

reference: https://github.com/SergeyPirogov/webdriver_manager
https://pypi.org/project/selenium/

Though after doing that cfg = Config() in autogpt/commands.py is undefined.

**Comment by phiroict** [comment]
Extra, after adding selenium and webdriver_manager (which the next one it fails on) I have all the modules but then the start of the process with `python -m ` returns this error: 

```
python -m autogpt
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/phiro/PycharmProjects/Auto-GPT/autogpt/__main__.py", line 12, in <module>
    from autogpt import commands as cmd
  File "/home/phiro/PycharmProjects/Auto-GPT/autogpt/commands.py", line 9, in <module>
    cfg = Config()
NameError: name 'Config' is not defined

```
I have pulled this version of autoGPT a month ago and did a git pull. 
Is the configuration also moved, I have adapted the new .env.template but that did not resolve it.

**Comment by pvanhemmen** [comment]
I know I can install missing dependencies myself, but they should all be part of the requirements.txt along with the correct versions to prevent errors like the one you are running into ;)

**Comment by phiroict** [comment]
Sure, but we want to help the developer by finding out as much as possible and try a few things to see we get it sorted then raise a PR or let the developer fix it themselves.

**Comment by phiroict** [comment]
Found it, in the `autogpt/commands.py` add this line to the top
```
from autogpt.config import Config
```
This will now start the process.

**Comment by draggingsurf** [comment]
after running all the above commands, this new error is coming--- Warning: The file 'auto-gpt.json' does not exist. Local memory would not be saved to a file.
![image](https://user-images.githubusercontent.com/125905102/232157405-bd80c6bd-2f4d-4cce-bea1-7634b62dbd67.png)

**Comment by wheresthecode** [comment]
> Found it, in the `autogpt/commands.py` add this line to the top
> 
> ```
> from autogpt.config import Config
> ```
> 
> Also add to the `requirement.txt file`
> 
> ```
> selenium
> webdriver_manager
> ```
> 
> This will now start the process. Added pull requests #1405 and #1409 for this.

Confirmed this as a fix

**Comment by Namontez23** [comment]
th

> > Found it, in the `autogpt/commands.py` add this line to the top
> > ```
> > from autogpt.config import Config
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Also add to the `requirement.txt file`
> > ```
> > selenium
> > webdriver_manager
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This will now start the process. Added pull requests #1405 and #1409 for this.
> 
> Confirmed this as a fix
 this fix worked for me. do the following to fix it. 
"Open the commands.py file located in the autogpt directory.

At the top of the file, add the following line:

arduino
Copy code
from autogpt.config import Config
This will import the Config class from the config.py file.

Save the commands.py file.

Open the requirements.txt file and add the following two lines:

Copy code
selenium
webdriver_manager
This will ensure that the selenium and webdriver_manager modules are installed and available for use.

Save the requirements.txt file."

**Comment by wadewilliams** [comment]
This is fixed in #1444 \close

**Comment by richbeales** [comment]
closing as superseded by v0.2.0
